src/Accordion/accordion.jsx

```diff
@@ -0,0 +1,138 @@
+/*
+  This file contains the Accordion component
+*/
+import React from 'react';
+import {
+  Accordion,
+  AccordionDetails,
+  AccordionSummary,
+  styled,
+} from '@mui/material';
+import PropTypes from 'prop-types';
+
+const StyledAccordion = styled(Accordion)(({ type }) => ({
+  ...(type === 'SheetReferences' && {
+    border: '1px solid #2b7b79',
+    boxShadow: 'none',
+  }),
+
+}));
+
+const StyledAccordionSummary = styled(AccordionSummary)(({ type, theme }) => ({
+  ...(type === 'NRCSAccordionSummary' && {
+    '&.MuiAccordionSummary-root': {
+      minHeight: '1.5rem',
+      padding: '0.3125rem 1rem',
+      backgroundColor: theme.palette.primary.dark,
+      color: theme.palette.primary.text,
+      '.MuiAccordionSummary-content': {
+        margin: '0',
+      },
+      '&.Mui-expanded': {
+        minHeight: '2rem',
+      },
+      '.MuiAccordionSummary-expandIconWrapper.Mui-expanded': {
+        transform: 'rotate(0deg) !important',
+      },
+    },
+  }),
+}));
+
+const StyledAccordionDetails = styled(AccordionDetails)(({ type, theme }) => ({
+  ...(type === 'NRCSAccordionDetails' && {
+    '&.MuiAccordionDetails-root': {
+      padding: ' 2%',
+      'th,td': {
+        color: theme.palette.primary.text,
+      },
+    },
+  }),
+}));
+
+export const PSAAccordion = ({
+  accordionType, defaultExpanded, expanded, accordionDataTest, onChange, accordionSx,
+  summaryContent, summaryExpandIcon, summarySx, summaryDataTest, summaryType, summaryTheme,
+  divider,
+  detailsContent, detailsDataTest, detailsType, detailsTheme,
+}) => {
+  return (
+    <StyledAccordion
+      type={accordionType}
+      defaultExpanded={defaultExpanded}
+      expanded={expanded || undefined}
+      data-test={accordionDataTest}
+      onChange={onChange}
+      sx={accordionSx}
+    >
+      <StyledAccordionSummary
+        sx={summarySx}
+        expandIcon={summaryExpandIcon}
+        data-test={summaryDataTest}
+        type={summaryType}
+        theme={summaryTheme}
+      >
+        {summaryContent}
+      </StyledAccordionSummary>
+      {divider && divider}
+      <StyledAccordionDetails
+        data-test={detailsDataTest}
+        type={detailsType}
+        theme={detailsTheme}
+      >
+        {detailsContent}
+      </StyledAccordionDetails>
+    </StyledAccordion>
+  );
+};
+
+/** PropTypes for better type checking */
+PSAAccordion.propTypes = {
+  /** Accordion Props */
+  /** The type of accordion */
+  accordionType: PropTypes.oneOf([
+    '',
+    'SheetReferences'
+  ]),
```

Comment:
Can you remove `''` from here and also in the stories because it doesn't make sense to have a `type=''`